examples/go-migrations/README.md

```diff
@@ -27,15 +27,10 @@ $
 
 ## Best practice: Split migrations into a standalone package
```

Comment:
It's pretty frustrating to not be able to run this example without `GOPATH` shenanigans and the `.sql` migration complicates this. I would just namespace everything here to `main` so the only expectation is for the user to get the `goose` library whether via `go get` or their own dependency manager.

```
.
├── 00001_create_users_table.sql
├── 00002_rename_root.go
├── custom.go
├── foo.db
├── goose
└── README.md
```